src/test/java/com/epam/training/center/hw1/DataProvider/IsPositiveDataProvider.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.training.center.hw1.DataProvider;
+
+import org.testng.annotations.DataProvider;
+
+public class IsPositiveDataProvider {
```

Comment:
According to java convention all that start from "is...." should return boolean. This class will mislead your colleagues. Pls rename, remove "is"